Core/SFContentBlockerManagerExtension.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  SFContentBlockerManagerExtension.swift
```

Comment:
ah sorry, it's because the name of the extension is different, i.e. `SFContentBlockerManager`. is that deliberate?